db/schema.rb

```diff
@@ -10,2437 +10,1932 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_05_04_153704) do
```

Comment:
Prettier got hold of the db/schema file. I'f added it to the ignore, but this one is pretty large as a result. I had tried to get it into two commits, but the first seemingly failed and I didn't notice.